numpy/_core/_add_newdocs.py

```diff
@@ -6937,19 +6937,19 @@ def refer_to_array_attribute(attr, method=True):
     array(["hello", "world"], dtype=StringDType())
 
     >>> arr = np.array(["hello", None, "world"],
-                       dtype=StringDType(na_object=None))
+    ...                dtype=StringDType(na_object=None))
     >>> arr
-    array(["hello", None, "world", dtype=StringDType(na_object=None))
+    array(["hello", None, "world"], dtype=StringDType(na_object=None))
     >>> arr[1] is None
     True
 
     >>> arr = np.array(["hello", np.nan, "world"],
-                       dtype=StringDType(na_object=np.nan))
+    ...                dtype=StringDType(na_object=np.nan))
     >>> np.isnan(arr)
     array([False, True, False])
 
     >>> np.array([1.2, object(), "hello world"],
-                 dtype=StringDType(coerce=True))
+    ...          dtype=StringDType(coerce=True))
```

Comment:
This is taken over from the initial version, but should this not be `coerce=False`? – see also https://github.com/numpy/numpy/blob/a1f2d582f84878b6c67bd641fa671a4cf868fe8c/doc/source/user/basics.strings.rst?plain=1#L185-L188

Reply:
Yes, but it seems already fixed.

Reply:
- Sorry, must have looked at the wrong branch then – in `main` and 2.2.x it's still `True`.

Reply:
Sorry, confusing that this is twice in the docs, and only once is wrong.  Yeah, should just fix it...

Reply:
I was a bit puzzled that it did not raise in doctests, but apparently the exception was quoted incompletely.

Reply:
As the likely author of the typo (didn't check) thanks for noticing and fixing 😀